README.md

```diff
@@ -6,9 +6,14 @@ This is a Python package that integrates Superset and CommCare HQ.
 Local Development
 -----------------
 
-Follow below instructions.
+### Preparing CommCare HQ
 
-### Setup env
+The 'User configurable reports UI' feature flag must be enabled for the
```

Comment:
Maybe more of an HQ discussion, but should we not have a different FF for this workflow so users don't have to be concerned about the `User Configurable Reports UI` feature flag until they actually want to user the UCRs?

Reply:
I think we can definitely pull CommCare Analytics into the current discussion around UCR feature flags.